service/DeliveryServiceModel.go

```diff
@@ -57,3 +61,48 @@ func (s *DeliveryService) UpdateDeliveryService(service dao.DeliveryService) err
 	}
 	return nil
 }
+
+func (s *DeliveryService) SaveLogoFile(cover []byte, id int) error {
+	ACCESS_KEY := os.Getenv("ACCESS_KEY")
+	SECRET_KEY := os.Getenv("SECRET_KEY")
+	endpoint := "fra1.digitaloceanspaces.com"
+	ssl := true
+
+	// Initiate a client using DigitalOcean Spaces.
+	client, err := minio.New(endpoint, ACCESS_KEY, SECRET_KEY, ssl)
```

Comment:
move connection initialization to a separate function so that if you need to store something else you don't need to read env variables in that function too